sumo_rl/environment/env.py

```diff
@@ -207,7 +246,88 @@ def _discretize_elapsed_time(self, elapsed):
         for i in range(self.max_green//self.delta_time):
             if elapsed <= self.delta_time + i*self.delta_time:
                 return i
-        return self.max_green//self.delta_time -1
+        return self.max_green//self.delta_time - 1
 
 
-    
+class SumoEnvironmentPZ(AECEnv, EzPickle):
+    metadata = {'render.modes': [], 'name': "sumo_rl_v0"}
+
+    def __init__(self, **kwargs):
+        EzPickle.__init__(self, **kwargs)
+        self._kwargs = kwargs
+
+        self.seed()
+
+        self.agents = self.env.ts_ids
+        self.possible_agents = self.env.ts_ids
+        self._agent_selector = agent_selector(self.agents)
+        self.agent_selection = self._agent_selector.reset()
+        # spaces
+        self.action_spaces = {a: self.env.action_spaces(a) for a in self.agents}
+        self.observation_spaces = {a: self.env.observation_spaces(a) for a in self.agents}
+
+        # dicts
+        self.rewards = {a: 0 for a in self.agents}
+        self.dones = {a: False for a in self.agents}
+        self.infos = {a: {} for a in self.agents}
+
+    def seed(self, seed=None):
+        #self.randomizer, seed = seeding.np_random(seed)
+        self.env = SumoEnvironment(**self._kwargs)
+
+    def reset(self):
+        self.env.reset()
+        self.agents = self.possible_agents[:]
+        self.agent_selection = self._agent_selector.reset()
+        self.rewards = {agent: 0 for agent in self.agents}
+        self._cumulative_rewards = {agent: 0 for agent in self.agents}
+        self.dones = {agent: False for agent in self.agents}
+        self.infos = {agent: {} for agent in self.agents}
+        return self.env.observations.copy()
```

Comment:
reset should not return anything

Reply:
In 'https://github.com/PettingZoo-Team/PettingZoo/blob/master/pettingzoo/utils/env.py':
    def reset(self):
        '''
        resets the environment and returns a dictionary of observations (keyed by the agent name)
        '''
        raise NotImplementedError
     This doc should be changed.

Reply:
Anyway, I removed it in https://github.com/LucasAlegre/sumo-rl/pull/45/commits/a39fdde0773eb8f5ed66796dca3531c6ce88e37b